assets/cubyz/blocks/lava.zig.zon

```diff
@@ -9,6 +9,9 @@
 	.collide = false,
 	.absorbedLight = 0x0084ff,
 	.emittedLight = 0xff7b00,
+	.density = 2.5,
+	.terminalVelocity = 1,
+	.mobility = 0.001,
```

Comment:
Hmm, this doesn't seem to do anything here. I'll make an issue for changing mobility for fluids/gases.